config/initializers/sequel.rb

```diff
@@ -2,7 +2,6 @@
 
 Sequel.split_symbols = true
 Sequel.extension :core_extensions, :postgres_schemata
-Sequel::Model.db.extension :pg_array, :pg_inet, :pg_hstore
```

Comment:
This is moved in to `config/application.rb` to allow us to set the extension conditionally (it requires a working database connection).